File: tmp/edit_admin.js
import fs from 'fs';
const filePath = 'c:\\Users\\muhsi\\OneDrive\\Documents\\Guardian Pharma APP\\src\\pages\\AdminDashboard.jsx';
let content = fs.readFileSync(filePath, 'utf8');

const target = '<option value="Maternity Care">Maternity Care</option>';
const replacement = target + '\n                                                     <option value="Surgical Products">Surgical Products</option>\n                                                     <option value="Physiotherapy">Physiotherapy</option>';

if (content.includes(target)) {
    content = content.replace(target, replacement);
    fs.writeFileSync(filePath, content, 'utf8');
    console.log("Success");
} else {
    console.log("Target not found");
}

File: tmp/edit_admin.py
import sys

file_path = r'c:\Users\muhsi\OneDrive\Documents\Guardian Pharma APP\src\pages\AdminDashboard.jsx'
with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

target = '<option value="Maternity Care">Maternity Care</option>'
replacement = target + '\n                                                     <option value="Surgical Products">Surgical Products</option>\n                                                     <option value="Physiotherapy">Physiotherapy</option>'

if target in content:
    new_content = content.replace(target, replacement)
    with open(file_path, 'w', encoding='utf-8') as f:
        f.write(new_content)
    print("Success")
else:
    print("Target not found")
